Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -918,6 +937,80 @@ final class VisibleItemsProvider {
     }
   }
 
+  private func handleMonthBackgroundItemsIfNeeded(
```

Comment:
```suggestion
  private func handleMonthBackgroundItemIfNeeded(
```

seems like we just build one item?

Reply:
We hand all visible ones, so if Jan, Feb, and Mar were fully or partially visible, this would make a month background for all 3. Basically, if there's an entry in `framesForVisibleMonths`, then that's one of the month's well make a background for.